lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -278,7 +278,7 @@ function dosomething_campaign_preprocess_hot_module(&$vars, &$wrapper) {
     $rb_percent = 0;
   }
 
-  $override_copy = variable_get('dosomething_campaign_progress_copy_override', NULL);
+  $override_copy = dosomething_helpers_get_variable('node', $vars['nid'], 'progress_copy');
```

Comment:
OR apparently I did change and I just forgot to save the file

<img width="1123" alt="screen shot 2015-08-31 at 11 28 01 am" src="https://cloud.githubusercontent.com/assets/897368/9582444/63516a4a-4fd3-11e5-9550-a72f14958955.png">